### PR TITLE
Car 96 auth flow

### PR DESCRIPTION
**Pull request by KariGarcia**

### CODE REVIEW

File: common/server.ts [hunk 1]
```diff
@@ -1,61 +1,63 @@
...
+};
+  
+const handleError = (error, dispatch) => {
+  if (error.response?.status === 401) {
+    console.log('Not authorized for execute this operation (401)');
+    handleUnauthorized(dispatch);
+  }
+  return {
+    status: error.response?.status,
+    message: error.response?.data?.message,
```
Comment (luferupa) [inline common/server.ts]: @KariGarcia just a question for me to understand. Here if the data is undefined what is assigned to the message?

File: common/api.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { updateToken } from "../stores/slices/userSlice";
+import localStorage from "./localStorage";
+
```
Comment (luferupa) [inline common/api.ts:3]: Is this file finally been used? Or can we delete it?

**Review by luferupa — APPROVED** [review]
So good!

File: common/api.ts (hunk 2 above)
Comment (KariGarcia) [reply common/api.ts:3]: yess, we are using the localStorage of the phone to store/get the token

File: common/server.ts (hunk 1 above)
Comment (KariGarcia) [reply common/server.ts]: nothing 😆, setting a default message

**Pull request merged**